export const saveErrorLogger = info => {
	console.log(info)
	return new Promise((resolve, reject) => {
		resolve()
	})
}
